### ISSUE TITLE
Provide support for client endpoint route

### ISSUE THREAD

**Issue by estein9825** [terraform-providers/terraform-provider-aws#7521]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Would like to have the ability to create a client endpoint route similar to the way AWS CLI offers create-client-vpn-route. This is especially helpful for creating access to a network or the internet

### New or Affected Resource(s)

* aws_ec2_client_vpn_route

### Potential Terraform Configuration

```hcl
resource "aws_ec2_client_vpn_route" "example" {
  description = "Internet access"
  client_vpn_endpoint_id = "${aws_ec2_client_vpn_endpoint.example.id}"
  subnet_id = "${aws_subnet.example.id}"
  cidr_block = "0.0.0.0/0"
}
```

### References

[CLI document](https://docs.aws.amazon.com/cli/latest/reference/ec2/create-client-vpn-route.html)

**Comment by yardensachs** [comment]
Should be closed, this is duplicated by #10437
And was closed by PR #14103
cc @bflad

**Comment by yardensachs** [comment]
Duplicate of #10437

**Comment by bflad** [comment]
This was previously released in [version 2.70.0 of the Terraform AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/v2.70.0/CHANGELOG.md) and has been available in all versions since. Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

For further feature requests or bug reports with this functionality, please create a [new GitHub issue](https://github.com/terraform-providers/terraform-provider-aws/issues/new/choose) following the template for triage. Thanks!
